rns_shell/platform/graphics/PlatformDisplay.h

```diff
@@ -67,6 +69,7 @@ class PlatformDisplay {
 
 private:
     static std::unique_ptr<PlatformDisplay> createPlatformDisplay();
+    static SkSize currentScreenDimension;
```

Comment:
Should the variable name be currentScreenSize? 
Becauss the name is not matching the get/set function name.

Reply:
sure